v63004/inref/gtm8874.m

```diff
@@ -107,17 +97,8 @@
 	WRITE "SET $ZGBLDIR=""otherB.gld"" ",!
 	SET $ZGBLDIR="otherB.gld"
 
-	WRITE "VIEW ""STATSHARE"":""DEFAULT"" ",!
-	VIEW "STATSHARE":"DEFAULT"
-	WRITE !
-	do checkRegion()
-	WRITE !
-
-	WRITE "VIEW ""NOSTATSHARE"":""DEFAULT"" ",!
-	VIEW "NOSTATSHARE":"DEFAULT"
 	WRITE !
```

Comment:
Now that we have switched to otherB.gld, we need to open the database here (e.g. set x=$get(^x)) and then invoke checkRegion() that way we find out if the otherB.dat that we opened just now had a .gst file open for it or not. And we need to do this for two cases where $VIEW("STATSHARE") is 1 and 0 at the time of switching to otherB.gld and have the output for both cases.